python/tests/test_swig_interface.py

```diff
@@ -403,3 +403,7 @@ def test_model_instance_settings_custom_x0(pysb_example_presimulation_module):
     assert model2.getInitialStateSensitivities() == sx0
     assert settings == amici.get_model_settings(model2)
 
+
+def test_solver_repr_doesnt_raise():
+    print(amici.CVodeSolver())
+    print(repr(amici.CVodeSolver()))
```

Comment:
isn't the default `__repr__` set by swig anyways (as for `self.this` anyway), so this wouldn't have raised even before these changes?